database/cloud architect/about.json

```
@@ -0,0 +1,23 @@
+[
+    {
+        "name": "What is a Cloud Architect?",
+        "description": "Unlock the Secrets of Cloud Architecture: Dive into Beginner-Friendly Resources Today!",
```

[Nitpick] This description could use more detail. Talk about how devs can benefit from this source based on your experience. 
```suggestion
        "description": "Unlock the Secrets of Cloud Architecture: Dive into Beginner-Friendly Resources Today!",
```